src/components/shared/transactionsTable/index.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { withTranslation } from 'react-i18next';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import Box from '../../toolbox/box';
+import TableRow from '../../toolbox/table/tableRow';
+import LiskAmount from '../liskAmount';
+import { DateTimeFromTimestamp } from '../../toolbox/timestamp';
+import Icon from '../../toolbox/icon';
+import AccountVisual from '../../toolbox/accountVisual';
+import styles from './transactionsTable.css';
+import IconlessTooltip from '../iconlessTooltip';
+
+class TransactionsTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sortingColumn: props.columns.find(column => column.defaultSort).key,
+      ascendingSorting: true,
+    };
+
+    this.changeSorting = this.changeSorting.bind(this);
+    this.renderCellContent = this.renderCellContent.bind(this);
+    this.renderTransactions = this.renderTransactions.bind(this);
+    this.loadMore = this.loadMore.bind(this);
+  }
+
+  renderCellContent(column, transaction) {
+    const { t } = this.props;
+
+    switch (column.key) {
+      case 'senderId':
+      case 'recipientId':
+        return (
+          <div className={`${styles.address}`}>
+            <AccountVisual address={transaction[column.key]} size={32} />
+            <span className={`${styles.addressValue}`}>{transaction[column.key]}</span>
+          </div>
+        );
+      case 'id':
+        return (
+          <Link to={`${column.pathPrefix}/${transaction[column.key]}`}>
+            {transaction[column.key]}
+          </Link>
+        );
+      case 'amount':
+        return (
+          <React.Fragment>
+            <LiskAmount val={transaction[column.key]} />
+&nbsp;
+            {t('LSK')}
+          </React.Fragment>
+        );
+      case 'fee':
+        return (
+          <IconlessTooltip
+            tooltipContent={<p>{`${t('Type')} ${transaction.type}`}</p>}
+            title={t('Transaction')}
+            className="showOnBottom"
+          >
+            <div>
+              <LiskAmount val={transaction[column.key]} />
+&nbsp;
+              {t('LSK')}
+            </div>
+          </IconlessTooltip>
+        );
+      case 'timestamp':
+        return <DateTimeFromTimestamp time={transaction[column.key]} token="LSK" />;
+      case 'confirmations':
+        return transaction.confirmations > 0 ? <Icon name="approved" /> : <Icon name="pending" />;
+      default:
+        return transaction[column.key];
+    }
+  }
+
+  changeSorting(sortingColumn) {
```

Comment:
There is another ticket for sorting (and filters): https://github.com/LiskHQ/lisk-hub/issues/2512

Reply:
Removed the sorting logic.